docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -534,6 +534,18 @@ A pipeline with only one single-threaded input may contribute up to 1.00, a pipe
 
 Additionally, some amount of back-pressure is both _normal_ and _expected_ for pipelines that are _pulling_ data, as this back-pressure allows them to slow down and pull data at a rate its downstream pipeline can tolerate.
 
+| `queue_persisted_growth_bytes` |
```

Comment:
@yaauie, I need your review these important lines. Want to be sure every single point we are going to deliver to the end-user shouldn't be missed. Should be concise!

Reply:
Sweet. I had been working on the docs and have just rebased and made a PR against your branch -> https://github.com/mashhurs/logstash/pull/3